### PR TITLE
FEAT(client): Add option to toggle TTS in settings menu 

### PR DESCRIPTION
**Pull request by BombJovi**
### Checks

- [✔] My commits follow the [commit guidelines](https://github.com/mumble-voip/mumble/blob/master/COMMIT_GUIDELINES.md)

Added option to enable/disable TTS from within settings menu.

Closes issue #5561

### CODE REVIEW

File: src/mumble/Log.ui [hunk 1]
```diff
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>580</width>
-    <height>496</height>
+    <width>649</width>
```
Comment (davidebeatrici) [inline src/mumble/Log.ui:9]: If the change in side is required (i.e. forced by Qt Creator), let's do 650.

File: src/mumble/Log.cpp [hunk 2]
```diff
@@ -238,6 +238,7 @@ void LogConfig::load(const Settings &r) {
 	qcbReadBackOwn->setChecked(r.bTTSMessageReadBack);
 	qcbNoScope->setChecked(r.bTTSNoScope);
 	qcbNoAuthor->setChecked(r.bTTSNoAuthor);
+    qcbEnableTTS->setChecked(r.bTTS);
```
Comment (davidebeatrici) [inline src/mumble/Log.cpp:241]: 4 spaces instead of a tab.

File: src/mumble/Log.cpp [hunk 3]
```diff
@@ -281,6 +282,7 @@ void LogConfig::save() const {
 	s.bTTSMessageReadBack = qcbReadBackOwn->isChecked();
 	s.bTTSNoScope         = qcbNoScope->isChecked();
 	s.bTTSNoAuthor        = qcbNoAuthor->isChecked();
+    s.bTTS                = qcbEnableTTS->isChecked();
```
Comment (davidebeatrici) [inline src/mumble/Log.cpp:285]: 4 spaces instead of a tab.

File: src/mumble/MainWindow.cpp [hunk 4]
```diff
@@ -350,7 +350,7 @@ void MainWindow::setupGui() {
 	updateAudioToolTips();
 
 #ifdef USE_NO_TTS
-	qaAudioTTS->setChecked(false);
+    qaAudioTTS->setChecked(false);
```
Comment (davidebeatrici) [inline src/mumble/MainWindow.cpp:353]: 4 spaces instead of a tab.

File: src/mumble/MainWindow.cpp [hunk 5]
```diff
@@ -2129,6 +2129,7 @@ void MainWindow::on_qmConfig_aboutToShow() {
 	qmConfig->addAction(qaAudioWizard);
 	qmConfig->addAction(qaConfigCert);
 	qmConfig->addSeparator();
+    qaAudioTTS->setChecked(Global::get().s.bTTS);
```
Comment (davidebeatrici) [inline src/mumble/MainWindow.cpp:2132]: 4 spaces instead of a tab.

File: src/mumble/Log.ui [hunk 6]
```diff
@@ -127,7 +99,20 @@
         </property>
        </widget>
       </item>
-      <item row="1" column="1">
+      <item row="0" column="0">
+       <widget class="QCheckBox" name="qcbEnableTTS">
```
Comment (davidebeatrici) [inline src/mumble/Log.ui:103]: What about simply `qcbTTS`?

**Review by davidebeatrici — APPROVED** [review]
Only cosmetic stuff, the code itself looks good to me!

File: src/mumble/Log.ui (hunk 6 above)
Comment (Krzmbrzl) [reply src/mumble/Log.ui:103]: I actually think `qcbEnableTTS` is more accurate and should therefore be preferred.

**Review by Krzmbrzl — CHANGES_REQUESTED** [review]
As already written on IRC: The commits need to be adjusted to match our commit guidelines and the code needs to be formatted using clang-format 10. If you don't have that, I can do the formatting for you.

Furthermore, you'll have to run the script `scripts/updatetranslations.py` to update the translation source strings so that they can be properly translated on our external platform (Weblate).

Note that in the end you should end up with only 2 commits:
1. The commit that makes the changes to add the new checkbox in the settings (that should be a plain `FEAT(client): ...` commit as it adds a feature, but doesn't change an existing one (the old checkbox for TTS in the dropwdown continues to exist) what would be implied by `FEAT/CHANGE`
2. The commit created by the translation script. No changes necessary for that one (the scripts will commit it in the proper format already)

That means that after you have taken care of the formatting, you squash all your commits into a single one, reword the commit message for that to match our guideline and then you can run the script for the translations. After that you perform a `git pull -f` to update your remote branch and with that, this PR :)

**Comment by BombJovi** [comment]
Big thank you to Krzmbrzl, davidebeatrici and Terry137 from the Matrix channel for guiding me through this process! I won't lie, git is pretty confusing to me still, I hope to understand it more in time. Thank you and let me know if there is anything else I need to change!

**Review by Krzmbrzl — APPROVED** [review]

**Comment by Krzmbrzl** [comment]
Kinda forgot about this PR... Sorry for that. Thanks for working on this, though :+1:

**Comment by BombJovi** [comment]
For sure! Thank you for code reviewing and assisting me with this pull request!

**Pull request merged**